test/php/gutenberg-coding-standards/Gutenberg/Sniffs/Commenting/FunctionCommentSinceTagSniff.php

```diff
@@ -0,0 +1,162 @@
+
+		$wrapping_tokens_to_check = array(
+			T_CLASS,
+			T_INTERFACE,
+			T_TRAIT,
+		);
+
+		foreach ( $wrapping_tokens_to_check as $wrapping_token_to_check ) {
+			if ( false !== $phpcsFile->getCondition( $stackPtr, $wrapping_token_to_check, false ) ) {
+				// This sniff only processes functions, not class methods.
+				return;
+			}
+		}
+
+		$missing_since_tag_error_message = sprintf( '@since tag is missing for the "%s()" function.', $function_name );
+
+		// All these tokens could be present before the docblock.
+		$tokens_before_the_docblock = array(
+			T_PUBLIC,
+			T_PROTECTED,
+			T_PRIVATE,
+			T_STATIC,
+			T_FINAL,
+			T_ABSTRACT,
+			T_WHITESPACE,
+		);
+
+		$doc_block_end_token = $phpcsFile->findPrevious( $tokens_before_the_docblock, ( $stackPtr - 1 ), null, true, null, true );
+		if ( ( false === $doc_block_end_token ) || ( T_DOC_COMMENT_CLOSE_TAG !== $tokens[ $doc_block_end_token ]['code'] ) ) {
+			$phpcsFile->addError( $missing_since_tag_error_message, $stackPtr, 'MissingSinceTag' );
+			return;
+		}
+
+		// The sniff intentionally doesn't check if the docblock has a valid open tag.
+		// Its only job is to make sure that the @since tag is present and has a valid version value.
+		$doc_block_start_token = $phpcsFile->findPrevious( Tokens::$commentTokens, ( $doc_block_end_token - 1 ), null, true, null, true );
+		if ( false === $doc_block_start_token ) {
+			$phpcsFile->addError( $missing_since_tag_error_message, $stackPtr, 'MissingSinceTag' );
+			return;
+		}
+
+		// This is the first non-docblock token, so the next token should be used.
+		++$doc_block_start_token;
+
+		$since_tag_token = $phpcsFile->findNext( T_DOC_COMMENT_TAG, $doc_block_start_token, $doc_block_end_token, false, '@since', true );
+		if ( false === $since_tag_token ) {
+			$phpcsFile->addError( $missing_since_tag_error_message, $stackPtr, 'MissingSinceTag' );
+			return;
+		}
+
+		$version_token = $phpcsFile->findNext( T_DOC_COMMENT_WHITESPACE, $since_tag_token + 1, null, true, null, true );
+		if ( ( false === $version_token ) || ( T_DOC_COMMENT_STRING !== $tokens[ $version_token ]['code'] ) ) {
+			$phpcsFile->addError( $missing_since_tag_error_message, $since_tag_token, 'MissingSinceTag' );
+			return;
+		}
+
+		$version_value = $tokens[ $version_token ]['content'];
+
+		if ( version_compare( $version_value, '0.0.1', '>=' ) ) {
+			// Validate the version value.
+			return;
+		}
+
+		$phpcsFile->addError(
+			'Invalid @since version value for the "%s()" function: "%s". Version value must be greater than or equal to 0.0.1.',
+			$version_token,
+			'InvalidSinceTagVersionValue',
+			array(
+				$function_name,
+				$version_value,
+			)
+		);
+	}
+
+	/**
+	 * Checks if the current package is experimental.
+	 *
+	 * @param File $phpcsFile The file being scanned.
+	 * @return bool Returns true if the current package is experimental.
+	 */
+	private static function is_experimental_package( File $phpcsFile ) {
```

Comment:
I think the name should get updated to:
```suggestion
	private static function is_experimental_block( File $phpcsFile ) {
```

Individual blocks only can be experimental, the block library package is stable.

Reply:
I agree, I've used some vague terminology here. Fixed in 0a615920215e8a4fc71a734992afa59406920def.